conf/modules.config

```diff
@@ -1,253 +1,1030 @@
 /*
 ========================================================================================
-    Config file for defining DSL2 per module options
+    Config file for defining DSL2 per module options and publishing paths
 ========================================================================================
     Available keys to override module options:
-        args            = Additional arguments appended to command in module.
-        args2           = Second set of arguments appended to command in module (multi-tool modules).
-        args3           = Third set of arguments appended to command in module (multi-tool modules).
-        publish_dir     = Directory to publish results.
-        publish_by_meta = Groovy list of keys available in meta map to append as directories to "publish_dir" path
-                            If publish_by_meta = true                 - Value of ${meta['id']} is appended as a directory to "publish_dir" path
-                            If publish_by_meta = ['id', 'custompath'] - If "id" is in meta map and "custompath" isn't then "${meta['id']}/custompath/"
-                                                                        is appended as a directory to "publish_dir" path
-                            If publish_by_meta = false / null         - No directories are appended to "publish_dir" path
-        publish_files   = Groovy map where key = "file_ext" and value = "directory" to publish results for that file extension
-                            The value of "directory" is appended to the standard "publish_dir" path as defined above.
-                            If publish_files = null (unspecified)     - All files are published.
-                            If publish_files = false                  - No files are published.
-        suffix          = File name suffix for output files.
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
```

Comment:
Should we add here the `task.ext.singularity_pull_docker_container` since I think now this is not mentioned anywhere or at least, I did not spot it. I suddenly realized when I saw the `singularity_pull_docker_container` removed from the ` nextflow_schema.json`

Reply:
Just saw that @mahesh-panchal requested to update the README.md so then it is fine!

Reply:
Misread this 🙈